packages/labs/analyzer/src/lib/analyzer.ts

```diff
@@ -63,14 +82,29 @@ export class Analyzer {
     for (const fileName of rootFileNames) {
       modules.push(this.analyzeFile(fileName as AbsolutePath));
     }
-    return new Package(modules);
+    return new Package({
+      modules,
+      tsConfig: this.commandLine,
+      packageJson: this.packageJson,
+    });
   }
 
   analyzeFile(fileName: AbsolutePath) {
     const sourceFile = this.program.getSourceFile(fileName)!;
+    const sourcePath = absoluteToPackage(fileName, this.packageRoot);
+    const fullSourcePath = path.join(this.packageRoot, sourcePath);
+    const jsPath = ts
```

Comment:
We should validate this in some of the tests.

Reply:
Added one